### PR TITLE
overthebox: Change tunnel tracking policy

### PR DESCRIPTION
**Pull request by Tikiwinkie**
Signed-off-by: Kevin Lanvin <kevin.lanvin@corp.ovh.com>

### CODE REVIEW

File: overthebox/files/bin/otb-action-configure [hunk 1]
```diff
@@ -88,16 +88,39 @@ case "$tun_app" in
...
+config_foreach delete_tracking_conf track
+case "$tun_app" in
+	glorytun|glorytun_hybrid)
+		tracked_interfaces=$(_get glorytun_conf | jq -r '@text "\(.dev) \(.additional_interfaces[].dev)"');;
+	glorytun_mud)
+		tracked_interfaces=$(_get glorytun_mud_conf | jq -r '@text "\(.dev)"');;
+esac
+for tracked_interface in $tracked_interfaces; do
+	uci -q batch <<-EOF
+	set "otb-tracker.$tracked_interface=track"
```
Comment (gregdel) [inline overthebox/files/bin/otb-action-configure]: I don't think you need double quotes here

**Comment by Tikiwinkie** [comment]
Useless quotes removed

File: overthebox/files/bin/otb-action-configure [hunk 2]
```diff
@@ -98,6 +98,30 @@ case "$tun_app" in
 		;;
 esac | uci -q batch
 
+delete_tracking_conf() {
+	generator=
+	config_get generator "$1" generated_by
```
Comment (PouuleT) [inline overthebox/files/bin/otb-action-configure]: Here you're looking for generated_by

File: overthebox/files/bin/otb-action-configure [hunk 3]
```diff
@@ -98,6 +98,30 @@ case "$tun_app" in
...
+	glorytun|glorytun_hybrid)
+		tracked_interfaces=$(_get glorytun_conf | jq -r '@text "\(.dev) \(.additional_interfaces[].dev)"');;
+	glorytun_mud)
+		tracked_interfaces=$(_get glorytun_mud_conf | jq -r '@text "\(.dev)"');;
+esac
+for tracked_interface in $tracked_interfaces; do
+	uci -q batch <<-EOF
+	set otb-tracker.$tracked_interface=track
+	set otb-tracker.$tracked_interface.tries=5
+	set otb-tracker.$tracked_interface.generatedby=$ME
```
Comment (PouuleT) [inline overthebox/files/bin/otb-action-configure]: Here you're setting generatedby
-> This cannot work

Comment (PouuleT) [inline overthebox/files/bin/otb-action-configure]: Anyway, line 24 of the file, all the config generatedby "$ME" is already deleted, including your tracker config

Comment (Tikiwinkie) [inline overthebox/files/bin/otb-action-configure]: Fixed : tracker config deletion has been removed since line 24 already does it

**Comment by gregdel** [comment]
Version bump ? :)

**Comment by Tikiwinkie** [comment]
Version bumped ;)

**Pull request merged**